api/endpoints/community.py

```diff
@@ -13,14 +12,14 @@ def make_blueprint(con):
     
     metric = CommunityMetrics(con)
     metric_rbu = RentBurdenedMetrics(con)
-    metric_tt = TaxiTripMetrics(con)
```

Comment:
Add this back, you are accidentally deleting the Taxi metrics from this endpoint.